build/assets/custom-dictionary.txt

```diff
@@ -243,6 +243,8 @@ translocations
 trinucleotide
 tsv
 xCell
+telomerase
+Poly-A
```

Comment:
Removing this dictionary element and changing (in `content/03.methods.md`) to "poly-A", consistent with the rest of the document.
```suggestion
```
It turns out this was also causing the build error, as spelling words can't contain a `-`

Reply:
Sure, I will do the changes as suggested. Thanks.